SystemInformer/mainwnd.c

```diff
@@ -284,46 +284,161 @@ PPH_STRING PhMwpInitializeWindowTitle(
+
+
+        // 4.5 - Elevated why
+        //
+        ULONG ElevationType = PhGetOwnTokenAttributes().ElevationType;
+        if ((ElevationType == TokenElevationTypeFull || ElevationType == TokenElevationTypeDefault) && !isRunningAsSYSTEMUser && !isHighILOrGreater)
+        {
+            if (ElevationType == TokenElevationTypeFull)                                                                                                    // Token reports as Full (Linked-token) or Default (= No Linked Token, IE UAC is OFF), but Medium IL and lower
+                PhAppendStringBuilder2(&stringBuilder, L" (Elevation : Token UAC). ");
+            else if (ElevationType == TokenElevationTypeDefault)
+                PhAppendStringBuilder2(&stringBuilder, L" (Elevation : Token Admin). ");
+        }
+        else if (!(ElevationType == TokenElevationTypeFull || ElevationType == TokenElevationTypeDefault) && isRunningAsSYSTEMUser && !isHighILOrGreater)
+            PhAppendStringBuilder2(&stringBuilder, L" (Elevation : SYSTEM). ");                                                                             // If running as SYSTEM but have a Medium IL, consider this "Elevated". The Token Elevation attribute wont be set for NT Authority\System
+        else if (!(ElevationType == TokenElevationTypeFull || ElevationType == TokenElevationTypeDefault) && !isRunningAsSYSTEMUser && isHighILOrGreater)
+            PhAppendStringBuilder2(&stringBuilder, L" (Elevation : IL). ");                                                                                 // Token = Full (UAC)
+        else if ((ElevationType == TokenElevationTypeFull || ElevationType == TokenElevationTypeDefault) && (isRunningAsSYSTEMUser || isHighILOrGreater))
+            PhAppendStringBuilder2(&stringBuilder, L" (Elevated). ");
+
+
+        // 5. Driver connection verbosity
+        //
+        KPH_LEVEL driverLevel = KphLevelEx(FALSE);
+        BOOL isConnectedToDriver = driverLevel != KphLevelNone;
+        if (!isConnectedToDriver)
+            PhAppendStringBuilder2(&stringBuilder, L"  |  <No-Driver> ");
+        else
+            PhAppendStringBuilder2(&stringBuilder, L"  |   ");
+
+        // 5.5 - Driver Load type (Features + Security)
+        //
+        switch (driverLevel)
+        {
+        case KphLevelMax:
+            PhAppendStringBuilder2(&stringBuilder, L"Driver Connection: ++[Maximum]  ");
+            break;
+        case KphLevelHigh:
+            PhAppendStringBuilder2(&stringBuilder, L"Driver Connection: +[High]  ");
+            break;
+        case KphLevelMed:
+            PhAppendStringBuilder2(&stringBuilder, L"Driver Connection: ~[Medium]  ");
+            break;
+        case KphLevelLow:
+            PhAppendStringBuilder2(&stringBuilder, L"Driver Connection: -[Lower]  ");
+            break;
+        case KphLevelMin:
+            PhAppendStringBuilder2(&stringBuilder, L"Driver Connection: --[Minimal]  ");
+            break;
```

Comment:
I think this could maybe do with a bit more nuance, I would suggest something like:

```cpp
if (PhGetIntegerSetting(L"KsiEnable") && !PhStartupParameters.NoKph)
{
    switch (KphLevelEx(FALSE))
    {
    case KphLevelMax:
        PhAppendStringBuilder2(&stringBuilder, L" | Driver Connection: ++[Maximum]");
        break;
    case KphLevelHigh:
        PhAppendStringBuilder2(&stringBuilder, L" | Driver Connection: +[High]");
        break;
    case KphLevelMed:
        PhAppendStringBuilder2(&stringBuilder, L" | Driver Connection: ~[Medium]");
        break;
     case KphLevelLow:
        PhAppendStringBuilder2(&stringBuilder, L" | Driver Connection: -[Lower]");
        break;
    case KphLevelMin:
        PhAppendStringBuilder2(&stringBuilder, L" | Driver Connection: --[Minimal]");
        break;
    default:
        PhAppendStringBuilder2(&stringBuilder, L" | Driver Connection: None");
        break;
    }
}
else
{
    PhAppendStringBuilder2(&stringBuilder, L" | Driver Disabled ");
}
```

There is also `KphCommsIsConnected` but I think `KphLevelEx` is probably in telling you "Non" in the case above. Since lack of a connection will return `KphLevelNone`.